src/reanimated2/Hooks.js

```diff
@@ -417,7 +417,7 @@ function areDependenciesEqual(nextDeps, prevDeps) {
   var objectIs = typeof Object.is === 'function' ? Object.is : is;
 
   function areHookInputsEqual(nextDeps, prevDeps) {
-    if (prevDeps === null) return !1;
+    if (prevDeps === null || (prevDeps.length !== nextDeps.length)) return !1;
```

Comment:
Unrelated, why we have `!1` here and `!0` a little below?

Reply:
what do you mean
> Unrelated

?

Reply:
I mean, this is not part of the changes here. I was just curious why we don't use plain booleans here.

Reply:
I misunderstood you again today 😅 I thought you were telling the code is unrelated but I see you meant the comment was.